components/LanguagePicker.jsx

```diff
@@ -0,0 +1,74 @@
+import {useState} from "react";
+import DropDownPicker from "react-native-dropdown-picker";
+import styled from "styled-components/native";
+import {useFonts} from "expo-font";
+
+const LanguageBar = styled.View`
+  justifyContent: space-between;
+  flexDirection: row;
+  margin-top: 16px;
+  width: 100%;
+`;
+
+const DropdownItem = styled.View`
+  width: 43%;
+`;
+
+export const LanguagePicker = ({languageItems}) => {
+    const [inputLanguageOpen, setInputLanguageOpen] = useState(false);
+    const [outputLanguageOpen, setOutputLanguageOpen] = useState(false);
+    const [inputLanguageValue, setInputLanguageValue] = useState('uk');
+    const [outputLanguageValue, setOutputLanguageValue] = useState('en');
+    const [items, setItems] = useState([...languageItems]);
+
+    const onInputLanguageOpen = () => {
+        setOutputLanguageOpen(false);
+    };
+
+    const onOutputLanguageOpen = () => {
+        setInputLanguageOpen(false);
+    };
+
+    return (
+        <LanguageBar>
+            <DropdownItem>
+                <DropDownPicker
+                    style={{
+                        backgroundColor: '#FAFAFA',
```

Comment:
It will be better to have styles in a separated file.

Reply:
thx